<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdenarEstudios</class>
 <widget class="QMainWindow" name="OrdenarEstudios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Paciente: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregaEst">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Estudio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrint">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>76</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ImprimirOrden</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>250</width>
      <height>350</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>350</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>350</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>230</width>
          <height>330</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="Estudios"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="nombrePa">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>154</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tipo de estudio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tipoEst"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Estudio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nombreEst"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>258</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Recomendaciones:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="recomendacion"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
